pmm-app/src/pmm-qan/panel/components/Filters/Filters.tsx

```diff
@@ -38,46 +38,64 @@ export const FiltersContainer: FC<FiltersContainerProps> = ({
     }
   }, [selectedCheckboxes]);
 
+  const ShowAllButton = () => (
+    <Button
+      type="link"
+      className={styles.showAllButton}
+      onClick={() => showSetAll(!showAll)}
+      disabled={!selectedCheckboxes}
+      data-qa="qan-filters-show-selected"
+    >
+      {showAll ? 'Show Selected' : 'Show All'}
+    </Button>
+  );
+
+  const ResetButton = () => (
+    <Button
+      type="link"
+      htmlType="reset"
+      className={styles.resetButton}
+      data-qa="qan-filters-reset-all"
+      onClick={() => {
+        setFilter('');
+        showSetAll(true);
+      }}
+      disabled={!selectedCheckboxes}
+    >
+      Reset All
```

Comment:
Many added strings can be placed on Messages file.

Reply:
It's not new strings, but good point will move it all there